### PR TITLE
ran Tox modernizer

### PR DESCRIPTION
**Pull request by mraarif**
relevant JIRA issue [here](https://openedx.atlassian.net/browse/BOM-1508).
Adding python3.8 in travis and removing django version prior to django2.2

### CODE REVIEW

**Comment by mraarif** [comment]
`.travis.yml` needs to be updated according to changes in tox config.

**Comment by awais786** [comment]
@iamsobanjaved looking good please squash.

File: tox.ini [hunk 1]
```diff
@@ -1,96 +1,94 @@
...
-    django21: Django>=2.1,<2.2
-    django22: Django>=2.2,<2.3
-    drf39: djangorestframework<3.10.0
-    drf310: djangorestframework<3.11.0
-    drflatest: djangorestframework
-    -rrequirements/test.txt
-commands =
-    python -Wd -m pytest {posargs:-n 3}
+deps = 
+	django22: Django>=2.2,<2.3
```
Comment (jmbowman) [inline tox.ini:12]: Why did the whitespace change here?  I thought we'd been using 4-space indentation pretty consistently in tox.ini files.  Making it hard to scan and see what actually changed.

Comment (iamsobanjaved) [reply tox.ini:12]: script replaced 4-spaces indentation with tab, should i revert whitespace changes?

Comment (jmbowman) [reply tox.ini:12]: Yes, we definitely don't want tabs in these files; please update the script to use 4 space indentation instead.

Comment (mraarif) [reply tox.ini:12]: Correction: we didn’t add a leading tab, we should be able to update the constants DJANGO_22_DEPEDENCY and
DJANGO_30_DEPENDENCY to add leading spaces without making any other changes.

Comment (UsamaSadiq) [reply tox.ini:12]: As discussed in the group, the script does not make any change here. You can fix this by converting tabs to spaces in the file. In PyCharm, you can do this by selecting **edit->ConvertIndents->ToSpaces**.

**Comment by iamsobanjaved** [comment]
@mzulqarnain1 has opened a new #667

**Pull request closed without merging**